content/download.html.haml

```diff
@@ -9,45 +9,92 @@
   %body
     = partial('toptoolbar.html.haml')
     = partial('downloadbanner.html.haml')
-    = partial('platforms.html.haml',:title => 'Installing Jenkins', :steps => [{title:'Linux',iconclass:'icon-download',id:1,short:'Install Jenkins on Linux',href:'#linux'},{title:'Windows',iconclass:'icon-loop',id:2,short:'Install Jenkins on Windows',href:'#windows'},{title:'Mac OSX',iconclass:'icon-plug',id:3,short:'Install Jenkins on Mac OSX',href:'#osx'},{title:'Docker',iconclass:'icon-users',id:4,short:'Run Jenkins in a Docker Container',href:'#container'}])
 
   .container
     .row.body
       .col-md-1
       .col-md-8
-        %h2 Jenkins Installation
-
+        %h2 Jenkins Installation and Setup
         .doc-block
-          %h4 Introdution
-          %p Lorem ipsum dolor sit amet, per stet error laudem ei, cum autem vocent complectitur an. Agam possit dicunt duo ex, rationibus consectetuer vis no. In vix scaevola tacimates, mea constituam suscipiantur delicatissimi cu. Pri ut nibh antiopam, fierent repudiare mel ex, quo ne dico omnesque. Mel et perfecto concludaturque, eligendi efficiantur et qui. Brute dicat facilisi cu nam.
-          %p Possit commune scriptorem ex vix, augue semper torquatos an est. Qui eu consul meliore recteque, sit diam dictas mandamus id. Id nec oporteat accusamus, ad eleifend constituto nam. Sed no clita nusquam, eu facilis dolores eos. In omittam explicari contentiones his, has prima erant te, usu clita inimicus dissentiet ea. Iuvaret singulis qui ei, liber libris vel ea. Te sea solet eleifend, vix primis ornatus deseruisse ne.
-          %p Wisi lobortis pri ei. Maluisset aliquando instructior vel ea. Ut duo amet dicam. Dolorum mandamus accusamus cum no, vix ex hinc appellantur, vim bonorum copiosae euripidis ut.
-
+          %h3 Introduction
+          %p With the release of 2.0, the installation of Jenkins remains largely the same but the initial setup is different. After you have chosen your preferred OS or container for installation the first time you run Jenkins you will be presented with a setup wizard to properly configure Jenkins. Jenkins will not be accessible via the web interface or CLI until the initial setup is complete.
```

Comment:
Ugh, please don't write haml like this, it makes it very difficult to review and edit. Place the text content nested under the node, like:

```haml
%p
  This is a paragraph!
```